systemtest/src/test/java/io/strimzi/systemtest/log/LoggingChangeST.java

```diff
@@ -472,4 +472,212 @@ protected void tearDownEnvironmentAfterAll() {
+                .build();
+
+        kubeClient().getClient().configMaps().inNamespace(NAMESPACE).createOrReplace(configMap);
+        StatefulSetUtils.waitForNoRollingUpdate(kafkaName, kafkaPods);
+        assertThat("Kafka pod should not roll", StatefulSetUtils.ssHasRolled(kafkaName, kafkaPods), is(false));
+        TestUtils.waitFor("Verify logger change", Constants.GLOBAL_POLL_INTERVAL, Constants.GLOBAL_TIMEOUT,
+            () -> cmdKubeClient().execInPodContainer(KafkaResources.kafkaPodName(CLUSTER_NAME, 0),
+                        "kafka", "/bin/bash", "-c", "bin/kafka-configs.sh --bootstrap-server localhost:9092 --describe --entity-type broker-loggers --entity-name 0").out()
+                        .contains("kafka.authorizer.logger=ERROR"));
+
+        // log4j.appender.CONSOLE.layout.ConversionPattern is changed and thus we need RU
+        configMap = new ConfigMapBuilder()
+                .withNewMetadata()
+                .withName("external-cm")
+                .withNamespace(NAMESPACE)
+                .endMetadata()
+                .withData(Collections.singletonMap("log4j.properties", "log4j.appender.CONSOLE=org.apache.log4j.ConsoleAppender\n" +
+                        "log4j.appender.CONSOLE.layout=org.apache.log4j.PatternLayout\n" +
+                        "log4j.appender.CONSOLE.layout.ConversionPattern=%d{ISO8601} %p %m (%c) [%t]\n" +
+                        "log4j.rootLogger=INFO, CONSOLE\n" +
+                        "log4j.logger.org.I0Itec.zkclient.ZkClient=ERROR\n" +
+                        "log4j.logger.org.apache.zookeeper=ERROR\n" +
+                        "log4j.logger.kafka=ERROR\n" +
+                        "log4j.logger.org.apache.kafka=ERROR\n" +
+                        "log4j.logger.kafka.request.logger=WARN\n" +
+                        "log4j.logger.kafka.network.Processor=ERROR\n" +
+                        "log4j.logger.kafka.server.KafkaApis=ERROR\n" +
+                        "log4j.logger.kafka.network.RequestChannel$=ERROR\n" +
+                        "log4j.logger.kafka.controller=ERROR\n" +
+                        "log4j.logger.kafka.log.LogCleaner=ERROR\n" +
+                        "log4j.logger.state.change.logger=TRACE\n" +
+                        "log4j.logger.kafka.authorizer.logger=DEBUG"))
+                .build();
+
+        kubeClient().getClient().configMaps().inNamespace(NAMESPACE).createOrReplace(configMap);
+        StatefulSetUtils.waitTillSsHasRolled(kafkaName, kafkaPods);
+
+        TestUtils.waitFor("Verify logger change", Constants.GLOBAL_POLL_INTERVAL, Constants.GLOBAL_TIMEOUT,
+            () -> cmdKubeClient().execInPodContainer(KafkaResources.kafkaPodName(CLUSTER_NAME, 0),
+                        "kafka", "/bin/bash", "-c", "bin/kafka-configs.sh --bootstrap-server localhost:9092 --describe --entity-type broker-loggers --entity-name 0").out()
+                        .contains("kafka.authorizer.logger=DEBUG"));
+    }
```

Comment:
Can you please move these tests before the `@BeforeAll` (after the bridge logging test)?

Reply:
Do the STs have `BeforeAll` methods at the end of the class?

Reply:
They should have :smile: at least for me it's more readable. I think that some STs are little bit mixed, but I'm gonna fix this in some future PRs :)

Reply:
Or do you think that `BeforeAll` should be on the beginning of the class?